### ISSUE TITLE
virus positive

### ISSUE THREAD

**Issue by ricochico** [TehCheat/PoEHUD#117]
new download is marked positive by windows defender!

Trojan:Win32/Skeeyah.A!rfn

**Comment by 0houndog0** [comment]
agreed master is showing  positive

**Comment by Kronix420** [comment]
I've already taken the liberty of doing a LOT of research on this. Please reopen this issue so we can keep it organized.

https://github.com/TehCheat/PoEHUD/issues/113

**Comment by zaafar** [comment]
are you guys using x64 poehud or x32?

**Comment by ricochico** [comment]
using x64 but the virus detection sees a win 32 virus vov (windows defender x64 btw.)

after what kronix said im assuming its a false positive, connected to the randomized naming of the exe.
im not a programming guru but perhaps let the user pick a random name? 
and sorry for opening another issue but i didnt see the old thread cause it was under closed issues:P

**Comment by zaafar** [comment]
I have tried to make a virus free poehud.exe, basically I formatted my windows and installed everything from scratch. but this is only for x64. So if u guys wanna give it a shot (https://github.com/zaafar/PoEHUD).

Virus Total says 0 positives: https://www.virustotal.com/en/file/61a1aeb16122c2ce3ad7d2d8915ff533a98d3719921106db6b6232a34c25f932/analysis/1491201722/
You guys can scan it before running it and let me know if there are any positives.

**Comment by TehCheat** [comment]
My guess is we have a troll falsely reporting the exe as malicious.  Yours might be clean today, but it won't be for long.

**Comment by Greg36** [comment]
It must be someone reporting. Yesterday download was blocked on Chrome, today it's fine but Avast still detects it as MalvareRep and even when I add it to whitelist it catches it again as "Generic" probably due to name randomization.

**Comment by TehCheat** [comment]
yeah, someone is falsely reporting it as malicious.  Last exe was clean when uploaded, now shows up as flagged 4/60.  https://virustotal.com/en/file/0890df74d82b1add5a26a568804a283de13176626b4cdc65837c9601bcbcb378/analysis/1491577095/

**Comment by zaafar** [comment]
wow, that's so true. I see that too. once it was uploaded....it got 0 positives and now it have 4-6.

**Comment by Aquakor** [comment]
https://www.virustotal.com/en/file/429a54cbb3847452a5b221c7e83b34e70474049a7351afba472467bee39045b9/analysis/1491985647/ 11/60

**Comment by TehHammer** [comment]
If you're concerned about it, build it yourself.  It's a false positive. Some idiot is reporting it as malicious.

**Comment by TehCheat** [comment]
Another false positive.  Someone is reporting it as malicious.  Anyone that thinks it's an actual threat should look over the code, and build it themselves.

**Comment by darkdex52** [comment]
I'm pretty positive the compiled executable infected my computer (or tried to do it). I downloaded from this git as .zip and launched shortcut.bat and it immediately opened windows settings, tried to change CD-key(?), opened Group Policy settings as well as BitLocker settings. I saw a green command line do something to /system32 files. Avira did report it as Trojant crypt.

**Comment by osoyoosking** [comment]
na it working fine just downloaded fresh copy and installed and nothing detected 1 out of 1000s of poeple useing this,, there clearly clicking on something else or they already had a virus on there system and it infects all of there exe and new ones they get so ya i wouldnt worrie about it i cant compile shyt so ya im happy to ignore any false positives lol which i havnt gotten so...ty

**Comment by TehCheat** [comment]
It definitely throws a false positive on perfectly clean systems.  It just depends on the anti-virus.  The older the poe executable gets, the more virus scanners have the signature of the exe and the more virus scanners flag it falsely.  I scanned the tools folder once and got a virus flagged in there.  I believe Havc mentioned that the way it scrambles is flagged because viruses often scramble their names to make it harder to detect.  Regardless, I'm not taking out the exe.  Forcing people to compile is a major pain for them, even if it is way safer for them.

**Comment by darkdex52** [comment]
Then at least could you please re-compile the executable, TehCheat, as you're the project lead?

**Comment by vmv** [comment]
Don't run "shortcut.bat" as administrator, because it will open all of your system32 *.exe. Just run it normally like everyone else.

**Comment by TehCheat** [comment]
I rebuilt the exe, but if the same people download it with the same virus scanners, it'll get flagged exactly the same.

**Comment by Greg36** [comment]
@darkdex52 In windows search for 'Command Prompt', click right on it and select 'run as administrator' you will see that the working folder points to `C:\WINDOWS\system32`. The [shortcut.bat](https://github.com/TehCheat/PoEHUD/blob/x64/shortcut.bat) file launches all .exe files in the current directory. You run the shortcut.bat as administrator so you executed all .exe files in system32 directory. That is not a virus, that is just your mistake :)

**Comment by zaafar** [comment]
Wow that's something new. :) That's why I never run the .bat file. I run the real exe file with whatever name it has on it.

**Comment by TehCheat** [comment]
There has been at least 4 or 5 people that have done that in the oc thread.  It's a fairly user error.
